openwisp_firmware_upgrader/migrations/0002_default_permissions.py

```diff
@@ -0,0 +1,60 @@
+from django.db import migrations
+from django.contrib.auth.models import Permission
+
+from . import create_default_permissions
+
+
+
+def assign_permissions_to_groups(apps, schema_editor):
```

Comment:
this function can be shared between the two migrations, I will show you

Reply:
Please see https://github.com/openwisp/openwisp-firmware-upgrader/commit/079cb2644da56cf1191904becde2d5f5341ebd81